heudiconv/convert.py

```diff
@@ -523,6 +523,35 @@ def update_uncombined_name(
     return filename
 
 
+def update_multiorient_name(
+    metadata: dict[str, Any],
+    filename: str,
+) -> str:
+    if "acq-" in filename:
+        lgr.warning(
+            "Not embedding multi-orientation information as prefix already uses acq- parameter."
```

Comment:
```suggestion
            "Not embedding multi-orientation information as `%r` already uses acq- parameter.", filename
```

so user gets a clue on what you are warning about.